zeppelin/cli.py

```diff
@@ -19,9 +23,23 @@ def main():
     else:
         args.out_filename = 'knowledge'
 
-    zeppelin_converter = ZeppelinConverter(args.in_filename, args.out_filename,
-                                           directory, 'anonymous', 'N/A', 'N/A')
-    zeppelin_converter.convert()
+    try:
+        with open(args.in_filename, 'rb') as raw:
+            t = json.load(raw)
+            full_path = os.path.join(directory, args.out_filename + '.md')
+            with open(full_path, 'w') as fout:
+                if 'results' in t['paragraphs'][0]:
+                    zeppelin_converter = nc(args.in_filename, args.out_filename,
+                                            directory, 'anonymous', 'N/A', 'N/A')
+                else:
+                    zeppelin_converter = lc(args.in_filename, args.out_filename,
+                                            directory, 'anonymous', 'N/A', 'N/A')
+
+                zeppelin_converter.convert(t, fout)
+
+    except ValueError as err:
+        print(err)
```

Comment:
Catching and printing the exception doesn't add much value compared to letting the exception raise. Could you instead print a human-readable message ?